app/services/ConsignmentService.scala

```diff
@@ -62,4 +64,11 @@ class ConsignmentService @Inject()(val graphqlConfiguration: GraphQLConfiguratio
     sendApiRequest(getConsignmentSummaryClient, getConsignmentSummary.document, token, variables)
       .map(data => data.getConsignment.get)
   }
+
+  def getConsignmentRef(consignmentId: UUID, token: BearerAccessToken): Future[getConsignmentReference.GetConsignment] = {
```

Comment:
Is this function needed? It looks like the existing `getConsignmentTransferSummary` function will return the consignment reference value as part of its response. That would save creating the new function and the new client: `getConsignmentReferenceClient`

Reply:
Good point. We discussed this with Suzanne and doing it this way will make it slightly quicker.

Reply:
OK. Fair enough.